src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -471,39 +468,81 @@ def _post_process(
     upload_env_key: str,
     workspace_id: int,
     load_id: str,
-    obj_tuple: _WSObjTuple,
     upload_dir: str,
```

Comment:
upload_dir is kind of a confusing name, maybe collections_source_dir?

Reply:
👍

Reply:
:+1: